.github/workflows/ci.yml

```diff
@@ -19,6 +19,8 @@ jobs:
         uses: dtolnay/rust-toolchain@stable
       - name: Checkout
         uses: actions/checkout@v4
+      - name: Check typos
+        uses: crate-ci/typos@master
```

Comment:
Let's have this as a separate job named `typos` since we don't need to install Rust toolchain for it.

Reply:
Done in 119e4e1.